celestia.vcxproj

```diff
@@ -95,7 +95,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <ClCompile>
-      <AdditionalIncludeDirectories>.\src;C:\tools\vcpkg\installed\x86-windows\include\cspice;C:\tools\vcpkg\installed\x86-windows\include\luajit;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>.\src;$(VCPKG_INSTALLATION_ROOT)\installed\x64-windows\include\cspice;$(VCPKG_INSTALLATION_ROOT)\installed\x64-windows\include\luajit;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
I think we should use macros from https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration instead of VCPKG_INSTALLATION_ROOT which is only specified on GitHub runners.

Reply:
I despise msbuild